src/components/molecule/tokenAmountInput/index.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import { NumericFormat } from "react-number-format";
+import Button from "../../atom/Button";
+
+type TokenAmountInputProps = {
+  tokenText: string;
+  onClick: () => void;
+  disabled?: boolean;
+  className?: string;
+  tokenIcon?: React.ReactNode;
+  isFocused?: boolean;
+  tokenValue?: number;
+  onSetIsFocused: (focus: boolean) => void;
+  onSetTokenValue: (value: number) => void;
+  ref: React.RefObject<HTMLDivElement>;
+};
+
+const TokenAmountInput = React.forwardRef(function TokenAmountInput(
+  {
+    tokenIcon,
+    tokenText,
+    disabled,
+    isFocused,
+    tokenValue,
+    onSetTokenValue,
+    onSetIsFocused,
+    onClick,
+  }: TokenAmountInputProps,
+  ref: React.ForwardedRef<HTMLDivElement>
+) {
+  return (
+    <div
+      ref={ref}
+      className={`relative flex h-[103px] w-[424px] items-center justify-start gap-2 rounded-[8px] bg-[#F3F5FB] px-4 py-6 ${
```

Comment:
we use classname package for all cases that we have conditions in className.

Reply:
done